utils.js

```diff
@@ -1,3 +1,6 @@
+const { readFileSync } = require('fs')
+const aws = require('aws-sdk');
```

Comment:
I believe this version of the aws-sdk is end of life, so we should really start using v3. [Here's a recent example of a simple drop-in kms-helper script using v3](https://github.com/NYPL/discovery-api/blob/qa-node20/lib/kms-helper.js)

Reply:
Fixed